OasisDEX/oasis-borrow
# Lw/3886 add trigger basic buy

# [Add BasicBuyTrigger from Basic Buy Form](https://app.shortcut.com/oazo-apps/story/3886/adding-trigger-from-basic-buy-form)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- Added transaction adding basic buy trigger on basic buy form
  
## How to test 🧪
  <Please explain how to test your changes>
- Run the dapp with goerli
- go to any vault
- go to optimization tab
- click confirm button (values are not yet read from form)
- successfully submit transaction that adds basic buy sell trigger
![image](https://user-images.githubusercontent.com/6871923/175525952-1993daec-d7c2-4c04-9cb3-cd2ab25edc13.png)

trigger should be successfully added https://goerli.etherscan.io/tx/0x3b360fe116bd65bf6ac856f555e78474bed192e36ac3df8bba8e22083eb92a33#eventlog
